### PR TITLE
Added current and target tooltip values on display

### PR DESCRIPTION
**Pull request by Elkrival**
Display tooltip values on hover.
In postProcessData the label property was changed to valueLabel. The label property overrides the value in the tooltip.

Screen Shot 2022-08-01 at 10 58 56 AM

### CODE REVIEW

File: server/controllers/chartsController.js [hunk 1]
```diff
@@ -7,18 +7,19 @@ const postProcessData = (data) => {
 
   Object.entries(data).forEach((entry) => {
     const [key, count] = entry
-    const [study, label, color] = key.split('-')
+    const [study, valueLabel, color, studyTarget] = key.split('-')
     const newEntry = {
       color,
       count,
-      label,
+      valueLabel,
```
Comment (mikestone14) [inline server/controllers/chartsController.js:14]: do we need to change anything on the front-end with this change?

File: server/controllers/chartsController.js [hunk 2]
```diff
@@ -7,18 +7,19 @@ const postProcessData = (data) => {
...
       study,
+      studyTarget,
```
Comment (mikestone14) [inline server/controllers/chartsController.js:16]: should we parse this string to an integer?

File: server/controllers/chartsController.js (hunk 1 above)
Comment (Elkrival) [reply server/controllers/chartsController.js:14]: The label key is unused by us on the front end

File: views/components/Graphs/BarGraph.jsx [hunk 3]
```diff
@@ -40,6 +41,15 @@ const BarGraph = ({ graph }) => {
                 fill: ({ datum }) => datum.color,
               },
             }}
+            labels={({ datum: { count, studyTarget } }) =>
+              `Current: ${count} \n Target: ${studyTarget}`
```
Comment (Elkrival) [inline views/components/Graphs/BarGraph.jsx:45]: https://github.com/AMP-SCZ/dpdash/pull/117/files#r934763025
It's used here as a template string, I could parse it I don't know if we have to. What do you think?

Comment (mikestone14) [reply views/components/Graphs/BarGraph.jsx:45]: I think it's fine as is. We might want it to be an integer when we need to convert to percentages and factor the target in, but we can cross that bridge when we get to it.

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**